### ISSUE TITLE
details of partition of NUS-WIDE

### ISSUE THREAD

**Issue by iTomxy** [HYPJUDY/caffe-dnnh#6]
hello, I'm asking a details of the partition of the NUS-WIDE data set: how do you randomly select 100 samples in each classes for the query set (and so does the training set)?
I mean, as it's a multi-label data set, a same very sample might be selected several times during the sampling of each classes.
thanks

**Comment by HYPJUDY** [comment]
Hi, it seems that I used the triplet CIFAR-10 dataset but not NUS-WIDE dataset. Where do you find the partition method of NUS-WIDE dataset? Thanks.

**Comment by iTomxy** [comment]
the paper used NUS-WIDE, I thought you might have dealt with it too, so...

**Comment by HYPJUDY** [comment]
No, I didn't. My work is limited to [these aspects](https://github.com/HYPJUDY/caffe-dnnh#my-work). Thanks.
